src/ruby/lib/grpc/generic/active_call.rb

```diff
@@ -44,7 +44,7 @@ def check_status
         # raise BadStatus, propagating the metadata if present.
         md = status.metadata
         with_sym_keys = Hash[md.each_pair.collect { |x, y| [x.to_sym, y] }]
```

Comment:
btw, the metadata value should be a multi-map (and it looks like a ruby Hash isn't)